<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>486</width>
         <height>372</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>769</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>目录树</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>视图</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="act_1"/>
   <addaction name="act_2"/>
   <addaction name="act_3"/>
   <addaction name="act_4"/>
   <addaction name="separator"/>
   <addaction name="act_5"/>
   <addaction name="act_6"/>
   <addaction name="act_7"/>
   <addaction name="act_8"/>
   <addaction name="act_9"/>
   <addaction name="separator"/>
   <addaction name="act_10"/>
   <addaction name="act_11"/>
   <addaction name="separator"/>
   <addaction name="act_12"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeWidget" name="treeWidget_2">
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="act_1">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FolderOpen"/>
   </property>
   <property name="text">
    <string>添加目录</string>
   </property>
   <property name="toolTip">
    <string>添加文件目录</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="act_2">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FolderNew"/>
   </property>
   <property name="text">
    <string>添加文件</string>
   </property>
   <property name="toolTip">
    <string>添加文件图片</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="act_3">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/qrc/制作特殊照片1.png</normaloff>:/qrc/制作特殊照片1.png</iconset>
   </property>
   <property name="text">
    <string>删除节点</string>
   </property>
   <property name="toolTip">
    <string>删除图片</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="act_4">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/qrc/为操作词配图片2.png</normaloff>:/qrc/为操作词配图片2.png</iconset>
   </property>
   <property name="text">
    <string>遍历节点</string>
   </property>
   <property name="toolTip">
    <string>遍历所有图片和目录</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="act_5">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/qrc/为操作词配图片3.png</normaloff>:/qrc/为操作词配图片3.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="toolTip">
    <string>放大图片</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="act_6">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/qrc/为操作词配图片4.png</normaloff>:/qrc/为操作词配图片4.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string>缩小图片</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="act_7">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/qrc/为操作配图片7.png</normaloff>:/qrc/为操作配图片7.png</iconset>
   </property>
   <property name="text">
    <string>按实际大小</string>
   </property>
   <property name="toolTip">
    <string>填满</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="act_8">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/qrc/为操作配图片8.png</normaloff>:/qrc/为操作配图片8.png</iconset>
   </property>
   <property name="text">
    <string>适合宽度</string>
   </property>
   <property name="toolTip">
    <string>原始图片高度</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="act_9">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/qrc/为操作配图片9.png</normaloff>:/qrc/为操作配图片9.png</iconset>
   </property>
   <property name="text">
    <string>适合高度</string>
   </property>
   <property name="toolTip">
    <string>原始高度</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="act_10">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/qrc/为操作词配图片5.png</normaloff>:/qrc/为操作词配图片5.png</iconset>
   </property>
   <property name="text">
    <string>窗口浮动</string>
   </property>
   <property name="toolTip">
    <string>窗口可自由拖动</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="act_11">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/qrc/为操作词配图片6.png</normaloff>:/qrc/为操作词配图片6.png</iconset>
   </property>
   <property name="text">
    <string>窗口可见</string>
   </property>
   <property name="toolTip">
    <string>显示图片</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="act_12">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::SystemLogOut"/>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出程序</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="qrc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>act_12</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
